protocol/streaming/full_node_streaming_manager.go

```diff
@@ -501,28 +483,37 @@ func (sm *FullNodeStreamingManagerImpl) TracksSubaccountId(subaccountId satypes.
 }
 
 func getStreamUpdatesFromOffchainUpdates(
-	offchainUpdates *clobtypes.OffchainUpdates,
+	v1updates []ocutypes.OffChainUpdateV1,
```

Comment:
Switch to `[]ocutypes.OffChainUpdateV1` so both it could be used by both [here](https://github.com/dydxprotocol/v4-chain/pull/2399/files#diff-368bd5a4ee5ee048eba2e17066095318806ae24044b9504c57da5380329e34d4R551) and [here](https://github.com/dydxprotocol/v4-chain/pull/2399/files#diff-368bd5a4ee5ee048eba2e17066095318806ae24044b9504c57da5380329e34d4R944)